examples/js/controllers.js

```diff
@@ -4058,12 +4058,12 @@ var app = angular.module('webapp');
             });
         }, 4000);
         angular.extend($scope, {
-            markersWatchOptions: {
-                doWatch: false,
-                isDeep: false,
-                individual: {
-                    doWatch: false,
-                    isDeep: false
+            watchOptions: {
+                markers: {
```

Comment:
IMHO, both should be allowed.  Then there is a way to be explicit if desired, but it isn't required.